### PR TITLE
Fixing typos and clarifying fork requirement

### PR DESCRIPTION
**Pull request by geckya**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,47 +1,48 @@
...
+This is a playground open to anyone looking to practice using git & GitHub in a group project. Feel free to make any and all changes as you test your GitHub chops. One caveat, we ask you please do not change the main readme unless you are making an improvement (P.S. please help the next person and make improvements).
 
-To reiterate this repository is to get people (you!) comfortable with using github do not be scared to try things out!
+To reiterate this repository is to get people (you!) comfortable with using GitHub - do not be scared to try things out!
 
 #### Give it a try!
 
-* [Clone] (https://help.github.com/articles/cloning-a-repository/) from your command line `git clone https://github.com/Data4Democracy/github-playground.git`. Optional: [Fork](https://help.github.com/articles/fork-a-repo/) the repo first.
-* Move to main directory `cd github-playground``
+* [Fork](https://help.github.com/articles/fork-a-repo/) the repo (this step is optional if you have have "contributer" access to the repo).
```
Comment (mattgawarecki) [inline README.md]: "contributor"

File: README.md [hunk 2]
```diff
@@ -1,47 +1,48 @@
...
+  * `git commit -m "More formal greeting"`
   * Do another `git push origin <branch>` and changes show automatically show up in the PR (No need to open another PR).
 
-This only scratches the surface of what is possible with git and github. Feel free continue to use this repo when you need to test out (or document for others!) more complicated workflow.
+This only scratches the surface of what is possible with git and GitHub. Feel free continue to use this repo when you need to test out (or document for others!) more complicated workflow.
 
 #### Other resources
-* Github [desktop](https://desktop.github.com/) client if you prefer to work with GUI
+* GitHub [desktop](https://desktop.github.com/) client if you prefer to work with GUI
 * [Try github](https://try.github.io/levels/1/challenges/1)
```
Comment (mattgawarecki) [inline README.md]: "Try GitHub"

File: README.md [hunk 3]
```diff
@@ -1,47 +1,48 @@
...
 * Lifesaving advice from [ohshitgit](http://ohshitgit.com/)
 * Udacity [course](https://www.udacity.com/course/how-to-use-git-and-github--ud775)
 
 #### Encounter issues?
 * Do not worry, we have all been there!
 * Join #github-help channel on slack and ask for help.
```
Comment (mattgawarecki) [inline README.md]: "Slack"

**Comment by bstarling** [comment]
Sorry I missed this. Feel free to merge this repo. Open to **EVERYONE**

**Pull request merged**